1.4/Defs/NewGuns.xml

```diff
@@ -38,20 +37,15 @@ Cons: Small proprietary clip / Slow to aim</description>
                 <verbClass>CombatExtended.Verb_ShootCE</verbClass>
                 <hasStandardCommand>true</hasStandardCommand>
                 <defaultProjectile>Bullet_762x51mmNATO_FMJ</defaultProjectile>
-                <warmupTime>1.1</warmupTime>
+                <warmupTime>0.76</warmupTime>
                 <range>55</range>
-                <burstShotCount>4</burstShotCount>
+                <burstShotCount>5</burstShotCount>
                 <ticksBetweenBurstShots>4</ticksBetweenBurstShots>
                 <soundCast>Shot_AssaultRifle</soundCast>
                 <soundCastTail>GunTail_Medium</soundCastTail>
                 <muzzleFlashScale>9</muzzleFlashScale>
             </li>
         </verbs>
-		<modExtensions>
-		<li Class="CombatExtended.GunDrawExtension">
-			<DrawSize>1.3,1.3</DrawSize>
```

Comment:
This should probably be kept, almost every gun needs this extension as depending on attatchments, guns will shrink when held.